swagger/swagger.go

```diff
@@ -32,6 +32,12 @@ type Config struct {
 	// Optional. Default: ./swagger.json
 	FilePath string
 
+	// FileContent for the content of the swagger.json or swagger.yaml file.
+	// If provided, FilePath will not be read.
+	//
+	// Optional. Default: nil
+	FileContent []byte
```

Comment:
pls add something in the readme section for this new setting

Reply:
f0070f2